internal/symbolz/symbolz.go

```diff
@@ -63,10 +63,29 @@ func Symbolize(p *profile.Profile, force bool, sources plugin.MappingSources, sy
 	return nil
 }
 
+// Check whether path ends with one of the suffixes listed in
+// pprof_remote_servers.html from the gperftools distribution
+func hasLegacySuffix(path string) bool {
+	var suffixes = [...]string{
```

Comment:
You shouldn't need the `...`.

Reply:
It seemed logical to me to make this a fixed size array rather than a slice. I guess this does not matter much in practice, though.